service/routes.py

```diff
@@ -32,7 +32,13 @@ def index():
 @app.route("/recommendations", methods=["GET"])
 def list_recommendations():
     """Returns all of the recommendation"""
-    recommendations = Recommendation.all()
+    app.logger.info("Request for recommendations list")
+    recommendations = []
+    src_product_id = request.args.get("src_product_id")
+    if src_product_id:
+        recommendations = Recommendation.find_by_src_id(int(src_product_id))
+    else:
+        recommendations = Recommendation.all()
```

Comment:
How should we combined src_product_id and rec_product_id together in the same function?